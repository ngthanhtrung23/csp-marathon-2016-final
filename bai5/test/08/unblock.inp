......
.111.2
0034.2
..3455
6664.7
.....7
