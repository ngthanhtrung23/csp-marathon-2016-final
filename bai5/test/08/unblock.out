20
1 L 1
2 U 1
4 U 2
5 L 1
7 U 2
6 R 3
3 D 2
5 L 3
3 U 2
6 L 3
4 D 3
7 D 2
2 D 2
1 R 3
3 U 2
0 R 3
3 D 2
1 L 1
2 U 2
0 R 1

